### PR TITLE
enable different plans for different currencies

### PR DESCRIPTION
**Pull request by ScottDowne**

### CODE REVIEW

**Comment by ScottDowne** [comment]
R?

File: routes/index.js [hunk 1]
```diff
@@ -54,14 +54,15 @@ var routes = {
         amount: transaction.amount,
         stripeToken: transaction.stripeToken,
         email: transaction.email,
+        currency: transaction.currency,
         metadata: {
-          firstname: transaction.firstName,
-          lastname: transaction.lastName,
+          firstname: transaction.first,
```
Comment (jbuck) [inline routes/index.js]: I almost wonder if we shouldn't just be creating a Stripe customer object first...

Comment (ScottDowne) [inline routes/index.js]: It's required, and I do create the customer inside the function I send this data to.

https://github.com/ScottDowne/donate.mozilla.org/blob/batmans-got-plans/routes/stripe.js#L22

Comment (ScottDowne) [inline routes/index.js]: Sounds like though the order of things is not obvious or optimal maybe?

**Comment by ScottDowne** [comment]
This had a merge conflict, but it's fixed now.

R? @jbuck or @alicoding whoever gets here first?

**Pull request merged**